### PR TITLE
Fix parenthetical issue in citation

### PR DESCRIPTION
**Pull request by jt-mchorse**

### CODE REVIEW

**Comment by jt-mchorse** [comment]
close #55

File: site/gatsby-site/src/templates/cite.js [hunk 1]
```diff
@@ -1,38 +1,35 @@
...
 const forcedNavOrder = config.sidebar.forcedNavOrder;
 
-const GetCitation = ({nodes}) => {
+const GetCitation = ({ nodes }) => {
 
   let docs = [];
-  nodes.forEach(({node})=>docs.push(node));
+  nodes.forEach(({ node }) => docs.push(node));
+
+  console.log(docs)
```
Comment (smcgregor) [inline site/gatsby-site/src/templates/cite.js:22]: Leftover console logging?

**Comment by smcgregor** [comment]
I checked the preview and it is good. Just one change needed, then good to merge.

**Comment by jt-mchorse** [comment]
I removed the console.log, we are good to go.

**Pull request merged**